helm_deploy/values-preprod.yaml

```diff
@@ -21,7 +21,7 @@ env:
   JAVA_OPTS: "-Xmx512m"
   SPRING_SECURITY_OAUTH2_RESOURCESERVER_JWT_JWK_SET_URI: https://sign-in-preprod.hmpps.service.justice.gov.uk/auth/.well-known/jwks.json
   OAUTH_ROOT_URL: https://sign-in-preprod.hmpps.service.justice.gov.uk/auth/
-  COMMUNITY_ENDPOINT_URL: https://community-api-secure.perf.delius.probation.hmpps.dsd.io
+  COMMUNITY_ENDPOINT_URL: https://community-api-secure.pre-prod.delius.probation.hmpps.dsd.io
   ASSESSMENT_ENDPOINT_URL: https://offender-prprod.aks-live-1.studio-hosting.service.justice.gov.uk
```

Comment:
Is this one right?
If so I ❤️ it that we have
preprod
pre-prod
prprod
all to identify preprod
Can we call our service pre_prod to keep this trend going?